content/api-streamer/producer.textile

```diff
@@ -0,0 +1,38 @@
+---
+title: Producer
+section: api-streamer
+index: 67
+---
+
+A Producer of data uses the "API Streamer":https://ably.io/api-streamer to share live data streams with subscribers. Examples of realtime data streams include live sports scores, positioning updates for transport and logistics, or live data streamed from IoT sensors. These data streams can easily be published to the API Streamer via an Ably DSN app, which can be shared with "data consumers":/api-streamer/consumer. 
+
+You can use the API Streamer to "deploy":#deploy, "manage":#manage and "distribute":#distribute your data streams hosted using your DSN apps.
+
+h3(#deploy). Deploy
+
+Built on our globally-distributed, serverless "Data Stream Network":https://www.ably.io/platform, the API Streamer lets you reliably deploy your streaming APIs. Issues typically encountered when deploying realtime APIs such as scalability, bursts of usage, uptime guarantees, message ordering and data integrity, are all taken care of. 
+
+<!-- INSERT IMAGE -->
+
+h3(#manage). Manage
+
+Use API Streamer’s realtime native API Management toolkit to maximize the value of your realtime APIs. Streamer’s developer portal gives you security, analytics, stream transformation, as well as monetization and billing tools. Streamer helps turn data streams into revenue streams. 
```

Comment:
I don't know what you mean by "realtime native API management", can you explain?

Reply:
^ @remotekieran